packages/pigeon/platform_tests/ios_swift_unit_tests/ios/RunnerTests/PrimitiveTests.swift

```diff
@@ -0,0 +1,230 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import XCTest
+@testable import Runner
+
+class MockPrimitiveHostApi: PrimitiveHostApi {
+  func anInt(value: Int32) -> Int32 { value }
+  func aBool(value: Bool) -> Bool { value }
+  func aString(value: String) -> String { value  }
+  func aDouble(value: Double) -> Double { value }
+  func aMap(value: [AnyHashable : Any?]) -> [AnyHashable : Any?] { value }
```

Comment:
`[Key: Value]` (no white space after the key) per style guide.

Reply:
Fixed